parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -1185,10 +1187,15 @@ builtinLookup
   , ("Int.shiftRight", shri)
   , ("Int.trailingZeros", tzeroi)
   , ("Int.leadingZeros", lzeroi)
+  , ("Int.and", andi)
+  , ("Int.or", ori)
+  , ("Int.xor", xori)
+  , ("Int.complement", compli)
```

Comment:
These were missing previously, so programs that used these would crash with the new runtime. Fixed and added usages of these to the test harness.